apps/demo/config/vite/app.ts

```diff
@@ -1,6 +1,9 @@
 import { join, resolve } from 'path';
 import { defineConfig } from 'vite';
 import handlebars from 'vite-plugin-handlebars';
+// Vite config cannot import untranspiled ES modules
```

Comment:
So what is what you are importing here? 🙂

Reply:
I have explain this more in the comment.